### PR TITLE
YBLB 24 cards

### PR DESCRIPTION
**Pull request by Simisays**
Only archival wirl isn't working

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/archival_whorl.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Archival Whorl
+ManaCost:3 U U
+Types:Sorcery
+K:Gift
+SVar:GiftAbility:DB$ MakeCard | Name$ Rhystic Study | Defined$ Promised | GiftDescription$ a Rhystic Study
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/archival_whorl.txt:5]: Card isn't created
 @Northmoc thoughts ?

File: forge-gui/res/cardsfolder/upcoming/archival_whorl.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+A:SP$ ChangeZoneAll | Origin$ Graveyard,Hand | Destination$ Library | ChangeType$ Card.YouOwn | Shuffle$ True | SubAbility$ DBDraw | SpellDescription$ Shuffle your hand and graveyard into your library, then draw seven cards. If the gift wasn't promised, each other player also shuffles their hand and graveyard into their library, then draws seven cards.
+SVar:DBDraw:DB$ Draw | Defined$ You | NumCards$ 7 | SubAbility$ DBChangeZoneAllOpponents
+SVar:DBChangeZoneAllOpponents:DB$ ChangeZoneAll | ConditionPresent$ Card.PromisedGift | ConditionCompare$ EQ0 | Origin$ Graveyard,Hand | Destination$ Library | ChangeType$ Card.OppOwn | Shuffle$ True | SubAbility$ DBDrawBis
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/archival_whorl.txt:8]: Does this needs to be "cards **other** players own" ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/archival_whorl.txt:8]: yes

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/archival_whorl.txt:8]: you changed the wrong line

File: forge-gui/res/cardsfolder/upcoming/archival_whorl.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/archival_whorl.txt:5]: it works, but you probably want to move it to Battlefield instead...

File: forge-gui/res/cardsfolder/upcoming/awestruck_cygnet.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:Awestruck Cygnet
+ManaCost:W
+Types:Creature Bird Scout
+PT:2/1
+K:Starting intensity:0
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.YouCtrl+withFlying | IsPresent$ Card.Self+namedAwestruck Cygnet | TriggerZones$ Battlefield | Execute$ TrigIntensify | TriggerDescription$ Whenever a creature you control with flying enters, if this creature is named Awestruck Cygnet, cards you own named Awestruck Cygnet intensify by 1.
+SVar:TrigIntensify:DB$ Intensify | AllDefined$ Card.Self,Card.inZoneGraveyard+namedAwestruck Cygnet+YouOwn,Card.inZoneHand+namedAwestruck Cygnet+YouOwn,Card.inZoneLibrary+namedAwestruck Cygnet+YouOwn
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/awestruck_cygnet.txt:7]: wrong valid

File: forge-gui/res/cardsfolder/upcoming/buxton_decorated_host.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Name:Buxton, Decorated Host
+ManaCost:3 G W
+Types:Legendary Creature Rabbit Noble
+PT:4/4
+K:Convoke
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | IsPresent$ Creature.tapped+YouCtrl | Execute$ TrigSeek | TriggerDescription$ At the beginning of your end step, if you control a tapped creature, seek a nonland permanent card with mana value X or less, where X is the number of tapped creatures you control. Put that card onto the battlefield.
+SVar:TrigSeek:DB$ Seek | Type$ Permanent.nonLand+cmcLTX | RememberFound$ True | SubAbility$ DBPut
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/buxton_decorated_host.txt:7]: LE

File: forge-gui/res/cardsfolder/upcoming/charged_conjuration.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Name:Charged Conjuration
+ManaCost:3 R
+Types:Enchantment
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigAnimate | TriggerDescription$ At the beginning of your upkeep, instant and sorcery cards in your hand perpetually gain "This spell costs less to cast."
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/charged_conjuration.txt:4]: bad description

File: forge-gui/res/cardsfolder/upcoming/cindercone_smite.txt [hunk 6]
```diff
@@ -0,0 +1,7 @@
+Name:Cindercone Smite
+ManaCost:R
+Types:Sorcery
+A:SP$ DealDamage | ValidTgts$ Creature | TgtPrompt$ Select target creature | NumDmg$ 2 | SubAbility$ DBTreasure | SpellDescription$ CARDNAME deals 2 damage to target creature
+SVar:DBTreasure:DB$ Token | ConditionCheckSVar$ X | TokenScript$ c_a_treasure_sac | SpellDescription$ Then create a Treasure token if you weren't the starting player.
+SVar:X:Count$StartingPlayer.1.0
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/cindercone_smite.txt:6]: wrong logic

File: forge-gui/res/cardsfolder/upcoming/recruit_instructor.txt [hunk 7]
```diff
@@ -0,0 +1,9 @@
+Name:Recruit Instructor
+ManaCost:R W
+Types:Creature Mouse Warrior
+PT:1/1
+T:Mode$ AttackersDeclared | ValidAttackers$ Mouse.YouCtrl | Execute$ TrigDraft | TriggerZones$ Battlefield | TriggerDescription$ Whenever one or more Mice you control attack, draft a card from CARDNAME's spellbook.
+SVar:TrigDraft:DB$ Draft | Spellbook$ Angelfire Ignition,Barge In,Become Brutes,Boon of Safety,Cheeky House-Mouse,Crumb and Get It,Defiant Strike,Embercleave,Feather of Flight,Mabel's Mettle,Might of the Meek,Moment of Heroism,Unleash Fury,War Squeak
+T:Mode$ BecomesTarget | ValidTarget$ Card.Self | ValidSource$ SpellAbility.YouCtrl | TriggerZones$ Battlefield | FirstTime$ True | Execute$ TrigToken | TriggerDescription$ Valiant — Whenever Recruit Instructor becomes the target of a spell or ability you control for the first time each turn, create a 1/1 white Mouse creature token.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/recruit_instructor.txt:7]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/resourceful_collector.txt [hunk 8]
```diff
@@ -0,0 +1,12 @@
+Name:Resourceful Collector
+ManaCost:1 R G
+Types:Creature Raccoon Druid
+PT:3/3
+K:Haste
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigRandom | TriggerDescription$ At the beginning of your end step, a random non-Food permanent card in your graveyard perpetually becomes a Food artifact in addition to its other types and gains "{2}, Exile this artifact: You gain 3 life." You may play that card for as long as it's in your graveyard.
+SVar:TrigRandom:DB$ ChooseCard | Choices$ Permanent.nonFood+YouOwn | ChoiceZone$ Graveyard | AtRandom$ True | Amount$ 1 | SubAbility$ DBAnimate
+SVar:DBAnimate:DB$ Animate | Defined$ ChosenCard | Types$ Food,Artifact | Duration$ Perpetual | Abilities$ FoodSac | staticAbilities$ MayPlay | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/resourceful_collector.txt:8]: the static isn't gained

File: forge-gui/res/cardsfolder/upcoming/sandcloud_harbinger.txt [hunk 9]
```diff
@@ -0,0 +1,11 @@
+Name:Sandcloud Harbinger
+ManaCost:2 W
+Types:Creature Elemental Horse
+PT:2/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigConjure | TriggerDescription$
+When CARDNAME enters, conjure three cards named Sunscorched Desert into the top ten cards of each player's library at random.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sandcloud_harbinger.txt:6]: bad line

File: forge-gui/res/cardsfolder/upcoming/traverse_valley.txt [hunk 10]
```diff
@@ -0,0 +1,8 @@
+Name:Traverse Valley
+ManaCost:G
+Types:Sorcery
+K:Kicker:Forage 
+A:SP$ Seek | Type$ Land.nonBasic | RememberFound$ True | SubAbility$ DBChangeZone | SpellDescription$ Seek a nonbasic land card. 
+SVar:DBChangeZone:DB$ ChangeZone | Defined$ Remembered | Condition$ Kicked | Origin$ Hand | Destination$ Battlefield | SubAbility$ DBCleanup | SpellDescription$ If this spell was kicked, put that card onto the battlefield tapped.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/traverse_valley.txt:6]: tapped

File: forge-gui/res/cardsfolder/upcoming/resourceful_collector.txt (hunk 8 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/resourceful_collector.txt:8]: The mayplay ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/resourceful_collector.txt]: yea

File: forge-gui/res/cardsfolder/upcoming/charged_conjuration.txt (hunk 5 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/charged_conjuration.txt]: mb update Oracle too

**Pull request merged**